docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst

```diff
@@ -0,0 +1,104 @@
+.. _dataset-creation-aws-bucket-minimum-policy:
+
+Minimum AWS Policy requirements
+===============================
+
+It is possibly to make your access policy very restrictive and to even deny anyone with the correct IAM user credentials or role from outside of e.g your VPC or of a certain IP range from reading your data.
+
+The only hard requirement Lightly requires to properly work is `S3:ListBucket`.
+With this permission Lightly will only be able to list the filenames within your bucket but can't actually access the contents of your data. Only you will be able to access your data's content.
+
+.. warning::
+    The Lightly Worker will need to be running within the permissioned zone (e.g within your VPC or IP range) and will need the configuration flag `datasource.bypass_verify` set to `True`.
```

Comment:
Maybe mention when and where exactly `datasource.bypass_verify` has to be set. Should it be passed as argument when starting the worker or can it also be passed as part of the `worker_config` when scheduling a run?

Reply:
@guarin I tested it with passing it as config to a scheduled job. I am not sure if both work out of the box? can you enlighten me? It is added to the `docker.yml`.

Reply:
Looking at the code the `worker_config` overwrites the argument. So it should be set in `worker_config`.